<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetJdkPath</class>
 <widget class="QDialog" name="SetJdkPath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>74</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置Jdk路径</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LabelPath">
       <property name="text">
        <string>Jdk路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonScan">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>SetJdkPath</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SetJdkPath</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
